common/main/root.xml

```diff
@@ -5372,159 +5372,155 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		</personName>
 		<personName length="monogram" style="formal" order="surnameFirst">
-			<alias source="locale" path="../personName"/>
+			<alias source="locale" path="../personName[@length='long'][@usage='referring'][@style='formal'][@order='surnameFirst']"/>
 		</personName>
 		<personName length="monogram" style="informal" order="sorting">
-			<alias source="locale" path="../personName"/>
+			<alias source="locale" path="../personName[@length='long'][@usage='referring'][@style='formal'][@order='sorting']"/>
 		</personName>
 		<personName length="monogram" style="informal" order="givenFirst">
-			<alias source="locale" path="../personName"/>
+			<alias source="locale" path="../personName[@length='long'][@usage='referring'][@style='formal'][@order='givenFirst']"/>
 		</personName>
 		<personName length="monogram" style="informal" order="surnameFirst">
-			<alias source="locale" path="../personName"/>
+			<alias source="locale" path="../personName[@length='long'][@usage='referring'][@style='formal'][@order='surnameFirst']"/>
 		</personName>
 		<personName length="monogramNarrow" style="formal" order="sorting">
-			<alias source="locale" path="../personName"/>
+			<alias source="locale" path="../personName[@length='long'][@usage='referring'][@style='formal'][@order='sorting']"/>
 		</personName>
 		<personName length="monogramNarrow" style="formal" order="givenFirst">
-			<alias source="locale" path="../personName"/>
+			<alias source="locale" path="../personName[@length='long'][@usage='referring'][@style='formal'][@order='givenFirst']"/>
 		</personName>
 		<personName length="monogramNarrow" style="formal" order="surnameFirst">
-			<alias source="locale" path="../personName"/>
+			<alias source="locale" path="../personName[@length='long'][@usage='referring'][@style='formal'][@order='surnameFirst']"/>
 		</personName>
 		<personName length="monogramNarrow" style="informal" order="sorting">
-			<alias source="locale" path="../personName"/>
+			<alias source="locale" path="../personName[@length='long'][@usage='referring'][@style='formal'][@order='sorting']"/>
 		</personName>
 		<personName length="monogramNarrow" style="informal" order="givenFirst">
-			<alias source="locale" path="../personName"/>
+			<alias source="locale" path="../personName[@length='long'][@usage='referring'][@style='formal'][@order='givenFirst']"/>
 		</personName>
 		<personName length="monogramNarrow" style="informal" order="surnameFirst">
-			<alias source="locale" path="../personName"/>
+			<alias source="locale" path="../personName[@length='long'][@usage='referring'][@style='formal'][@order='surnameFirst']"/>
 		</personName>
 		<sampleName item="givenSurname">
 			<nameField type="prefix">Prefix</nameField>
 			<nameField type="given">Given</nameField>
-			<nameField type="given-informal">given-informal</nameField>
+			<nameField type="given-informal">givenInformal</nameField>
```

Comment:
Note the "givenInformal" here is just a dummy name, not an attribute or placeholder value.